interfaces/cython/cantera/cti2yaml.py

```diff
@@ -1723,7 +1760,34 @@ def main():
     else:
         output_file = pathlib.Path(args.output)
 
-    convert(input_file, output_file)
+    n_spc, n_rxn, surfaces, output_name = convert(
+        input_file, output_file, encoding=args.input_encoding)
+
+    if not args.quiet:
+        print(f"Wrote YAML mechanism file to '{output_name}'.")
+        print(f"Mechanism contains {n_spc} species and {n_rxn} reactions.")
+
+    # Do full validation by importing the resulting mechanism
+    if args.no_validate:
+        return
+
+    try:
+        import cantera as ct
+    except ImportError:
+        print("WARNING: Unable to import Cantera Python module. ")
+        print("Output mechanism has not been validated")
+        sys.exit(0)
```

Comment:
This is perhaps mirroring some other code, but I suspect that `sys.exit(1)` here would be more appropriate? Also, printing to `sys.stderr`?

Reply:
Other than not using `logger`, this is exactly equivalent of what `ck2yaml` does; as the script could be run without cantera deliberately, I think that may have been a deliberate choice.